app/views/layouts/admin.html.erb

```diff
@@ -64,27 +64,7 @@
     </div>
 
     <div id="footer" class="footer">
```

Comment:
could you move this out into the partial as well? would also be better as a `<footer>` element